mmv1/products/cloudrunv2/api.yaml

```diff
@@ -0,0 +1,717 @@
+            description: |-
+              A reference to a customer managed encryption key (CMEK) to use to encrypt this container image. For more information, go to https://cloud.google.com/run/docs/securing/using-cmek
+          - !ruby/object:Api::Type::Integer
+            name: "maxInstanceRequestConcurrency"
+            description: |-
+              Sets the maximum number of requests that each serving instance can receive.      
+      - !ruby/object:Api::Type::Array
+        name: "traffic"
+        description: |-
+          Specifies how to distribute traffic over a collection of Revisions belonging to the Service. If traffic is empty or not provided, defaults to 100% traffic to the latest Ready Revision.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+          - !ruby/object:Api::Type::Enum
+            name: "type"
+            description: |
+              The allocation type for this traffic target.
+            values:
+              - :TRAFFIC_TARGET_ALLOCATION_TYPE_LATEST
+              - :TRAFFIC_TARGET_ALLOCATION_TYPE_REVISION
+          - !ruby/object:Api::Type::String
+            name: "revision"
+            description: |
+              Revision to which to send this portion of traffic, if traffic allocation is by revision.
+          - !ruby/object:Api::Type::Integer
+            name: "percent"
+            description: |
+              Specifies percent of the traffic to this Revision. This defaults to zero if unspecified.
+          - !ruby/object:Api::Type::String
+            name: "tag"
+            description: |
+              Indicates a string to be part of the URI to exclusively reference this target.  
+      - !ruby/object:Api::Type::String
+        name: "observedGeneration"
+        output: true
+        description: |
+          The generation of this Service currently serving traffic. See comments in reconciling for additional information on reconciliation process in Cloud Run. Please note that unlike v1, this is an int64 value. As with most Google APIs, its JSON representation will be a string instead of an integer.
+      - !ruby/object:Api::Type::NestedObject
+        name: "terminalCondition"
+        output: true
+        description: |
+          The Condition of this Service, containing its readiness status, and detailed error information in case it did not reach a serving state. See comments in reconciling for additional information on reconciliation process in Cloud Run.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: "type"
+            output: true
+            description: |-
+              type is used to communicate the status of the reconciliation process. See also: https://github.com/knative/serving/blob/main/docs/spec/errors.md#error-conditions-and-reporting Types common to all resources include: * "Ready": True when the Resource is ready.
+          - !ruby/object:Api::Type::String
+            name: "status"
```

Comment:
Should this be `state` (here and under `conditions`)? I'm seeing state instead of status at https://cloud.google.com/run/docs/reference/rest/v2/Condition.